compiler/test/stdlib/string.test.gr

```diff
@@ -315,119 +318,137 @@ assert String.decode(String.encode(emojis, String.UTF32_BE), String.UTF32_BE) ==
 assert String.decode(String.encode(emojis, String.UTF32_LE), String.UTF32_LE) ==
   emojis
 
+let buf = Bytes.make(500)
+// ensure buffer is clear for our assertions on contents
+Bytes.clear(buf)
 // decodeRange
 // 51 is chosen to stress-test these functions, since it's not an aligned offset
 assert String.decodeRange(
-  String.encodeAt(emojis, String.UTF8, Bytes.make(500), 51),
+  String.encodeAt(emojis, String.UTF8, buf, 51),
```

Comment:
Why were these changed to re-use a Bytes?

Reply:
The GC actually cleans things up in a timely manner now, so if you allocate a `Bytes`, stop using it, and allocate another one of the same size immediately, you're pretty like to get the same memory region right back. It's definitely time to `calloc` these.